src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Attributes/OpenApiResponseWithBodyAttribute.cs

```diff
@@ -32,6 +33,11 @@ public OpenApiResponseWithBodyAttribute(HttpStatusCode statusCode, string conten
         /// </summary>
         public virtual Type CustomHeaderType { get; set; }
 
+        /// <summary>
+        /// Gets or sets a list of comma seperated additional content types.
+        /// </summary>
+        public virtual string AdditionalContentTypes { get; set; }
```

Comment:
We need to discuss this first:

> Introducing this additional property vs. Using the existing property, `ContentType`

I'm on the latter side, and your PR is the former side.

Reply:
I am fine with either reusing the `ContentType` or introducing a new `ContentTypes` property. 

Personally, I would then just re-use `ContentType` to reduce the hurdle to upgrade

Reply:
OK. Let's re-use the existing `ContentType` property for now.

Reply:
Done!